platform/distance.cpp

```diff
@@ -79,9 +84,10 @@ Distance Distance::CreateFormatted(double distanceInMeters)
 Distance Distance::CreateAltitudeFormatted(double meters)
 {
   Distance elevation = Distance(meters).To(
-      measurement_utils::GetMeasurementUnits() == measurement_utils::Units::Metric ? Units::Meters : Units::Feet);
-  if (elevation.IsLowUnits())
-    elevation.m_distance = WithPrecision(elevation.m_distance, 0);
+      GetMeasurementUnits() == measurement_utils::Units::Metric ? Units::Meters : Units::Feet);
+
+  ASSERT(elevation.IsLowUnits(), ());
```

Comment:
What's the reason of this assert? The value is already converted to lower units

Reply:
Probably, I added it by mistake. It's not needed here. Elevation will always be in lower units.
We call function with meters and convert them to meters or feet. There is no way it will be Km or Mi.
I think you can remove this assert

Reply:
We can leave it in case if logic above will change someday. At least this assert also works like documentation.